packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/host/dedicatedCloud-datacenter-host.service.js

```diff
@@ -12,4 +12,13 @@ export default class {
       },
     );
   }
+
+  getHostLocation(serviceName, datacenterId, hostId) {
+    return this.OvhHttp.get(
```

Comment:
use $http instead

Reply:
Didn't know that, was keeping the same coding style as other files, switching to $http